make/autoconf/jdk-version.m4

```diff
@@ -135,6 +135,19 @@ AC_DEFUN_ONCE([JDKVER_SETUP_JDK_VERSION_NUMBERS],
     PRODUCT_NAME="$with_product_name"
   fi
 
+  # The product suffix, if any
+  AC_ARG_WITH(product-suffix, [AS_HELP_STRING([--with-product-suffix],
+      [Set product suffix. Among other uses, appends the 'java.runtime.name' system property. @<:@not specified@:>@])])
```

Comment:
The help doesn't read well; suggest replacing 'appends' with 'contributes to'.

Reply:
updated